common/djangoapps/student/views/management.py

```diff
@@ -1340,3 +1341,37 @@ def text_me_the_app(request):
     }
 
     return render_to_response('text-me-the-app.html', context)
+
+
```

Comment:
Can we make this function as a task? Celery task to run in the background?

Reply:
done